docs/content/operator/validator-node/overview.mdx

```diff
@@ -3,19 +3,19 @@ title: Validator Node Overview
 tags: [node-operation]
 teams:
   - iotaledger/node
-description: Learn how to set up, configure, and manage an IOTA Validator node, including staking, reference gas price, and tallying rules.
+description: Learn how to set up, configure, and manage an IOTA validator node, including staking, reference gas price, and tallying rules.
 ---
 
 import StakingPoolReqs from "../../_snippets/staking-pool-reqs.mdx";
 import ValidatorRequirements from "../../_snippets/operator/validator-requirements-tab.mdx"
 
 Validators on the IOTA network run special nodes and have additional tasks and responsibilities beyond those of full node operators.
 
-## Requirements to Run a Validator on IOTA
+## Requirements to Run a Validator Node on IOTA
 
-To run an IOTA validator, you must set up and configure an IOTA Validator node. After you have a running node, you must have a minimum of 2 million IOTA in your staking pool to join the validator set on the IOTA network.
-
-To learn how to set up and configure an IOTA Validator node, see [Docker Setup](docker.mdx) or [Systemd Setup](systemd.mdx). The guide includes all of the information you need to configure your Validator node. It also provides guidance on the tasks you must perform after you join the validator set.
+To run an IOTA validator node, you must:
+1. Setup and run a validator node. See setup guides for [Docker](docker.mdx) or [Systemd](systemd.mdx).
+2. Get a minimum of **2 million IOTA** in your staking pool to join the validator set. This amount can be delegated to your node by other users.
```

Comment:
We should re-add the info that the guides include tasks that the validators MUST perform after they have the minimum stake. Maybe step 3?